RiotSharp/Properties/AssemblyInfo.cs

```diff
@@ -24,4 +24,4 @@
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("2.6.0.*")]
+//[assembly: AssemblyVersion("2.6.0")]
```

Comment:
Unfortunately, there's no "right way" for the versions that I'm aware of. It depends on the toolset you're using to build. How do you intend to generate version numbers during your build process?